packages/@webex/plugin-cc/test/unit/spec/cc.ts

```diff
@@ -1,348 +1,117 @@
-    
-      expect(resolveMock).not.toHaveBeenCalled();
-    
-      // Fast-forward until all timers have been executed
-      jest.runAllTimers();
-    
-      expect(rejectMock).toHaveBeenCalled();
-      clearTimeout(eventHandler.timeoutId);
-    
-      jest.useRealTimers();
+      await expect(webex.cc.stationLogin(options)).rejects.toThrow(error);
     });
-  
-    // it('should reject the promise if the event times out', (done) => {
-    //   const resolveMock = jest.fn();
-    //   const rejectMock = jest.fn().mockImplementation((error) => {
-    //     expect(error.message).toBe('Time out waiting for event: register');
-    //     done();
-    //   });
-  
-    //   webex.cc.addEventHandler('register', resolveMock, rejectMock);
-  
-    //   const eventHandler = webex.cc.eventHandlers.get('register');
-    //   expect(eventHandler).toBeDefined();
-  
-    //   // Simulate timeout
-    //   setTimeout(() => {
-    //     expect(rejectMock).toHaveBeenCalled();
-    //   }, WEBSOCKET_EVENT_TIMEOUT + 100);
-    // }, WEBSOCKET_EVENT_TIMEOUT + 200);
-  });
-
-  describe('#establishConnection', () => {
-    it('should establish WebSocket connection with correct parameters', async () => {
-      webSocketMock.subscribeAndConnect.mockResolvedValue();
 
-      webex.cc.register();
+    it('should handle error during stationLogin with BROWSER login option', async () => {
+      const options = {
+        teamId: 'teamId',
+        loginOption: LoginOption.BROWSER,
+      };
 
-      expect(webSocketMock.subscribeAndConnect).toHaveBeenCalled();
-      expect(webSocketMock.subscribeAndConnect).toHaveBeenCalledWith({
-        datachannelUrl: 'https://api.example.com/v1/notification/subscribe',
-        body: {
-          force: true,
-          isKeepAliveEnabled: false,
-          clientType: 'WebexCCSDK',
-          allowMultiLogin: true,
-        },
-      });
-    }, 1000);
+      webex.cc.agentConfig = {
+        agentId: 'agentId',
+      };
 
-    it('should log error and throw if connection fails', async () => {
-      const error = new Error('Connection error');
-      webSocketMock.subscribeAndConnect.mockRejectedValue(error);
-      webex.logger.info = jest.fn();
+      const error = new Error('Login failed');
+      const registerWebCallingLineMock = jest.fn().mockRejectedValue(error);
+      WebRTCCalling.prototype.registerWebCallingLine = registerWebCallingLineMock;
+      jest.spyOn(webex.cc.agent, 'stationLogin').mockRejectedValue(error);
 
-      try {
-        await webex.cc.establishConnection((err) => {
-          throw err;
-        });
-        assert.fail('Expected error was not thrown');
-      } catch (err) {
-        assert.equal(err, error);
-        expect(webex.logger.info).toHaveBeenCalled();
-        expect(webex.logger.info).toHaveBeenCalledWith(`Error connecting and subscribing: ${error}`);
-      }
-    }, 1000);
+      await expect(webex.cc.stationLogin(options)).rejects.toThrow(error);
```

Comment:
Tests for register function has been removed, please add them back. Please check the coverage report and cover all the conditions and functions